umap/static/umap/js/umap.features.js

```diff
@@ -606,17 +606,22 @@ L.U.Marker = L.Marker.extend({
 
   appendEditFieldsets: function (container) {
     L.U.FeatureMixin.appendEditFieldsets.call(this, container)
+    const latbk = this._latlng.lat,
```

Comment:
I can't see where you reuse these vars 🤔

Reply:
Nowhere, that's coming from a first attempt :p

Thanks for spotting! :)